lib/utils/export/normalize-line-break.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Normalize line break characters to CRLF
+ * @param content string Content to normalize
+ */
+const normalizeLineBreak = (content: string) =>
+  content.replace(/\r\n/g, '\n').replace(/\n/g, '\r\n');
```

Comment:
This could be done in one pass by using [negative lookbehind](https://javascript.info/regexp-lookahead-lookbehind#lookbehind) in the regular expression: `(?<!\r)\n`. Unfortunately this expression is [not supported yet in all browsers](https://caniuse.com/mdn-javascript_builtins_regexp_lookbehind_assertion) so it's not safe to use it.

Reply:
performance here won't be an issue.

Reply:
though if you wanted to do it in one RegExp operation, you can do that with an alternation group, though that's not likely to be faster

```ts
content.replace(/\r\n|\n/g, '\r\n');
```

this matches `\r\n` first and essentially leaves it the same, but if that match fails, it then matches on a single newline and adds the leading '\r'